doc/storage.md

```diff
@@ -258,6 +235,39 @@ lxc storage create pool2 dir source=/data/lxd
+:--                             | :---      | :--------                         | :------                    | :----------
+source                          | string    | -                                 | -                          | Path to block device or loop file or filesystem entry
+rsync.bwlimit                   | string    | -                                 | 0 (no limit)               | Specifies the upper limit to be placed on the socket I/O whenever rsync has to be used to transfer storage entities.
+rsync.compression               | bool      | appropriate driver                | true                       | Whether to use compression while migrating storage pools.
+volatile.initial\_source        | string    | -                                 | -                          | Records the actual source passed during creating (e.g. /dev/sdb).
+volatile.pool.pristine          | string    | -                                 | true                       | Whether the pool has been empty on creation time.
+volume.size                     | string    | appropriate driver                | unlimited (10GiB for block)| Default volume size
+ceph.cluster\_name              | string    | ceph driver                       | ceph                       | Name of the ceph cluster in which to create new storage pools.
+ceph.osd.force\_reuse           | bool      | ceph driver                       | false                      | Force using an osd storage pool that is already in use by another LXD instance.
+ceph.osd.pg\_num                | string    | ceph driver                       | 32                         | Number of placement groups for the osd storage pool.
+ceph.osd.pool\_name             | string    | ceph driver                       | name of the pool           | Name of the osd storage pool.
+ceph.osd.data\_pool\_name       | string    | ceph driver                       | -                          | Name of the osd data pool.
+ceph.rbd.clone\_copy            | string    | ceph driver                       | true                       | Whether to use RBD lightweight clones rather than full dataset copies.
+ceph.rbd.features               | string    | ceph driver                       | layering                   | Comma separate list of RBD features to enable on the volumes.
+ceph.user.name                  | string    | ceph driver                       | admin                      | The ceph user to use when creating storage pools and volumes.
+cephfs.cluster\_name            | string    | cephfs driver                     | ceph                       | Name of the ceph cluster in which to create new storage pools.
+cephfs.path                     | string    | cephfs driver                     | /                          | The base path for the CEPHFS mount
+cephfs.user.name                | string    | cephfs driver                     | admin                      | The ceph user to use when creating storage pools and volumes.
```

Comment:
The `cephfs` ones only apply to the cephfs driver so should be moved to that one.